tests/parameter_handler/prm/parameter_handler_read_json_04.json

```diff
@@ -0,0 +1,8 @@
+{
+    "test 0": "1",
+    "test 1": "1",
+    "test 2": "1",
+    "test 3": {
+        "test 4" : "1"
+    }
```

Comment:
I don't understand what is happening with this test. You only declare parameters `test 0` and `test 1`. How come we accept this as a valid input file? Is it because we read the whole file and then go through our own parameters to see which ones have been read, ignoring everything that has been read but does not correspond to our own parameters?

Reply:
Exactly. The ignoring works well for the parameters but not for the sections with the result that new sections are created.